### DISCUSSION TITLE
Vercel.json proxying the root url with "rewrites"

### DISCUSSION THREAD

**Discussion by marcellkiss** [vercel/community#27] [Help] [answered]
I have two applications:  
- A single page application (**myApp**), hosted on Vercel, 
- A wordpress site (**myWp**), hosted on a wordpress host 

My goal:
 1. **myhostname.com/app** should show **myApp**
 2. **myhostname.com** should show **myWp**

The first one is working, by adding these lines to vercel.json:
```
    {
      "source": "/app/:match*",
      "destination": "/:match*"
    },
```

Accomplishing the second one should look this:
```
    {
      "source": "/",
      "destination": "https://mywp.mywphost.com"
    },
```

But this rule doesn't take effect, seems like I can't proxy the root url, just subpaths.

Have you any recommendations, how to solve this?

**Comment by Pieparker** [comment, marked as answer]
It may be easier to think about this the other way around! Is it possible to delegate a path at your WP host?

Perhaps you can update the `destination` to be `https://mywp.mywphost.com/` or `https://mywp.mywphost.com/index.html`. If these don't work, I would assume that this is deliberate, in order to prevent a pattern of proxying that, more often than not, is used by bad actors!
